File: container_zad3_dotnet/zad3/Util/CityGen.cs
public class CityGen{
    private  string [] cities = new string[]{"RESEN","STRUMICA","KICEVO","OHRID","SKOPJE","VALANDOVO","MAVROVO"};
    private readonly Random random = new Random();
    public String getRandomCity(){
        return cities[random.Next(0, cities.Length)];
    }
}

File: zad2/src/main/java/internettehologii/individualnizadaci/zad2/services/CityGen.java
package internettehologii.individualnizadaci.zad2.services;

import java.util.Random;

public class CityGen{
    private final String [] cities = new String[]{"RESEN","STRUMICA","KICEVO","OHRID","SKOPJE","VALANDOVO","MAVROVO"};
    private final Random random = new Random();
    public String getRandomCity(){
        return cities[random.nextInt(cities.length)];
    }
}
